### PR TITLE
Feature/chat locker 6

### PR DESCRIPTION
**Pull request by douglasrapp**

### CODE REVIEW

File: client/stylesheets/base.less [hunk 1]
```diff
@@ -1748,11 +1748,24 @@ a.github-fork {
 			color: @primary-font-color;
 		}
 	}
+	.security-banner.U { color: white; background-color: green }
```
Comment (rwakida) [inline client/stylesheets/base.less]: maybe we can create variables for these in _variables.less and reference them here.  Then we can reuse them in other places.

**Comment by rwakida** [comment]
Looks good

File: client/stylesheets/base.less (hunk 1 above)
Comment (douglasrapp) [inline client/stylesheets/base.less]: Do you mean variables for the colors? Like, say, "@unclassified-color"?

Comment (douglasrapp) [inline client/stylesheets/base.less]: Done.

**Comment by douglasrapp** [comment]
Merging to master

**Pull request merged**